zeeguu/core/content_quality/quality_filter.py

```diff
@@ -46,8 +46,7 @@ def sufficient_quality_html(html):
 
 
 def sufficient_quality_plain_text(text):
-    word_count = len(text.split(" "))
-
+    word_count = len(text.split())
```

Comment:
in which way does this make a difference?

Reply:
The two are not equivalent. If no parameter is passed python looks for any consecuntive white spaces and consideres them as a single seperator:

https://stackoverflow.com/questions/62013468/is-there-a-difference-between-split-vs-split
```
>>> "I am    a    friend!".split()
['I', 'am', 'a', 'friend!']
>>> "I am    a    friend!".split(" ")
['I', 'am', '', '', '', 'a', '', '', '', 'friend!']
>>>
```

Reply:
whoa. didn't think about consecutive spaces. good catch @tfnribeiro !